cmd/ping/main.go

```diff
@@ -85,22 +85,25 @@ func serveHTTP(ctlConf config, h healthcheck.Handler) func() {
 	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		if _, err := w.Write([]byte(ctlConf.HTTPResponse)); err != nil {
 			w.WriteHeader(http.StatusInternalServerError)
-			logger.WithError(err).Error("error responding to http request")
+			logger.WithError(err).Error("Error responding to http request")
```

Comment:
Please revert all capitalisation back, it was done intentional here
https://github.com/golang/go/wiki/CodeReviewComments#error-strings
```suggestion
			logger.WithError(err).Error("error responding to http request")
```